rust/agents/relayer/src/relayer.rs

```diff
@@ -304,15 +308,12 @@ impl BaseAgent for Relayer {
             tasks.push(self.run_merkle_tree_processor(origin));
         }
 
-        run_all(tasks)
+        join_all(tasks).await;
```

Comment:
what's the difference with this change?

Reply:
`run_all` is a utility from our codebase to abort other tasks when one fails. It does the same thing as `try_join_all` afaict so I'll actually just remove it from the codebase.

`join_all` let's tasks run even if one panics, which I realised we shouldn't do, so I replaced most call sites of `join_all` with `try_join_all` and a panic.